Makefile

```diff
@@ -54,14 +56,31 @@ test-race:                      ## Run tests with race detector.
 test-cover:                     ## Run tests and collect coverage information.
 	go test $(TEST_FLAGS) -coverprofile=cover.out -covermode=count ./...
 
-check-license:                  ## Check that all files have the same license header.
+check:                          ## Run required checkers and linters.
 	go run .github/check-license.go
+	go-sumtype ./vendor/... ./...
```

Comment:
why don't we run `golangci-lint` here?

Reply:
https://golangci.com runs it on CI.
During development, the most typical use is `golangci-lint run` which is not much longer than `make check`. But another common use-case is `golangci-lint run --new-from-rev=PMM-2.0` to check only new/changed code. I kind of want to promote both without introducing three Makefile targets (the third one for Travis CI that does not run `golangci-lint`).

Reply:
Another option is
```
check-travis:
  go run .github/check-license.go
  go-sumtype ./vendor/... ./...

check: check-travis
  golangci-lint run
```

But that leaves out `golangci-lint run --new-from-rev=PMM-2.0`.